src/clj/rems/css/styles.clj

```diff
@@ -272,21 +272,51 @@
                      :color "#ccc"}
                     [:&:hover {:color (util/get-theme-attribute :color4)
                                :background-color "#eee"}]]]
-   [:.alert-info
-    (s/descendant :.state-info :.phases :.phase.completed)
-    {:color (util/get-theme-attribute :info-color)
-     :background-color (util/get-theme-attribute :info-bgcolor)}]
+
+   [:text-primary {:color (util/get-theme-attribute :text-primary :alert-primary-color)}]
```

Comment:
`:text-primary` refers to the `<text-primary>` element which doesn't exists. `:.text-primary` would refer to the class.